packages/eas-cli/src/build/graphql.ts

```diff
@@ -25,11 +25,13 @@ export function transformProjectArchive(archiveSource: ArchiveSource): ProjectAr
     return {
       type: ProjectArchiveSourceType.S3,
       bucketKey: archiveSource.bucketKey,
+      metadataLocation: archiveSource.metadataLocation,
```

Comment:
```suggestion
```
We are no longer uploading archive sources to S3 AFAIK, so I believe we can delete it